tutorials/io/files/resource_queue.gd

```diff
@@ -1,6 +1,9 @@
+extends Node
+
 var thread
 var mutex
-var sem
+var semaphore
+var exit_thread := false
```

Comment:
We don't use type hints in code samples in the documentation (for consistency with other pages):

```suggestion
var exit_thread = false
```